translations/tiled_de.ts

```diff
@@ -3899,7 +3900,7 @@ Bitte Format angeben.</translation>
     <message>
         <location line="+1"/>
         <source>Erase Terrain</source>
-        <translation type="unfinished"></translation>
+        <translation>Terrain löschen</translation>
```

Comment:
Hmm, because it still uses the Terrain Brush rather than being a separate tool. That way you can also capture "no terrain" from the map and erase with it. The button is below the list in a way that I thought it would be suggesting a similar mode of operation, rather than to delete the selected terrain. But, I can see how it could be somewhat confusing. Anyway, not going to change for Tiled 0.16 anymore so maybe you have a more appropriate German word here?